pangeo-deploy/values.yaml

```diff
@@ -94,6 +117,12 @@ homeDirectories:
 # * ingress-nginx.controller.service.loadBalancerIP: ...
 
 prometheus-operator:
+  alertmanager:
+    enabled: false
+
+  nodeExporter:
```

Comment:
Do you know what nodeExporter does? This does per-*node* monitoring? But we're mostly interested in per-*pod* statistics?

Reply:
Node exporters should be looking at hardware / OS (if I'm reading this right): https://github.com/prometheus/node_exporter#node-exporter, so yeah, per-node monitoring. We have only really done per-pod stuff up to now I think.

Reply:
Sounds good, thanks.